.gitlab-ci.yml

```diff
@@ -312,7 +308,7 @@ build:edge+flambda:dune:dev:
 
 build:base+async:
   extends: .build-template
-  stage: stage-1
+  stage: build
```

Comment:
You can just remove this line, right? Since it extends `.build-template`.

Reply:
indeed